src/main/resources/application.yml

```diff
@@ -37,6 +37,7 @@ queueconfig:
   action-scheduler-routing-key-uac: event.uac.*
   action-scheduler-routing-key-case: event.case.*
   refusal-routing-key: event.respondent.refusal
+  receipt-routing-key: Receipt.Responses.binding
```

Comment:
Should this be `event.response.receipt` like what Liam suggested in [field pr](https://github.com/ONSdigital/census-rm-fieldwork-adapter/pull/4/files#r312414457)?

Reply:
That was the binding I was previously  told to use

Reply:
I'll change it